src/main/resources/config.yml

```diff
@@ -108,240 +110,263 @@ whole-number-prices-only: false
   enchance-shop-protect: false
 
-  #/qs find <itemName> command options
-  #This command lets users shop quickly without wasting time searching
+  #/qs find <itemName> command settings.
+  #This command lets users shop quickly without wasting time searching.
   finding:
-    #Should we use the old logic? (looking for closet shop and let player look at it)
+    #Should QS use the old logic? (looking for closest shop and let players look at it)
     oldLogic: false
-    #How far should we search in blocks?
+    #How far should QS search? (in blocks)
     distance: 45
     #How many nearby shop should be displayed?
     limit: 10
-    #Including non-loaded shops?
+    #Should non-loaded shops be incuded in the search?
     all: true
-    #Check out-of-stock/space shop, may hurt the performance
+    #Should QS exclude out-of-stock/space shops? 
+    #This may improve performance.
     exclude-out-of-stock: false
 
-  #If vault doesn't return a coins symbol, QuickShop will use this symbol to replace it.
+  #If vault doesn't return a currency symbol, QuickShop will use this symbol.
   alternate-currency-symbol: '$'
 
-  #This symbol setting for multi-currency supports
-  #If no any hit in this list, will fallback to the name of currency it self
+  #This setting is for multi-currency setups.
+  #If nothing matches in this list, it falls back to the name of currency itself.
   alternate-currency-symbol-list:
     - CNY;¥
     - USD;$
 
-  #It should fix Vault or an economy plugin that is not correctly processing the price and returns a "0" result problem.
-  #If you turn on this option, plugin will force use "alternate-currency-symbol" option.
+  #This should fix Vault or an economy plugin incorrectly processing the price and returning a "0" result.
+  #If you enable this setting, then QS will force use the "alternate-currency-symbol".
   disable-vault-format: false
 
-  #If show currency symbol on the right side
-  #By default, is on the left side
-  #Only work when disable-vault-format is true
+  #If the currency symbol should be displayed on the right side (eg 1234$).
+  #By default, it is on the left side (eg $1234).
+  #Only works when disable-vault-format is true.
   currency-symbol-on-right: false
 
-  #Ignore trade messages if the shop is unlimited
+  #Ignore trade messages if the shop is unlimited.
   ignore-unlimited-shop-messages: false
 
-  #Allow auto fetching player shops, if you disable this, players need to use /qs fetchmessage to fetch the shops history messages. (Disabling it may cause database issues.)
+  #Allow auto fetching of player shops.
+  #If you disable this, then the players need to use "/qs fetchmessage" to fetch the shops history messages. 
+  #Disabling it may cause database issues!
   auto-fetch-shop-messages: true
 
-  #Ignore cancel chat event, can improve compatibility with other chat plugin in some cases.
+  #Ignore cancel chat event, can sometimes improve compatibility with some chat plugins.
```

Comment:
cancel->cancelled

Reply:
ohh i thought "cancel chat event" was what its name is...